pages.zh/linux/chage.md

```diff
@@ -0,0 +1,24 @@
+# chage
+
+> 更改用户账户和密码到期信息。
+> 更多信息：<https://manned.org/chage>.
+
+- 列出用户的密码信息：
+
+`chage -l {{用户名}}`
+
+- 启用密码在10天内过期：
```

Comment:
Almost done, thank you. Please place one space before/after English words and numbers.
```suggestion
- 启用密码在 10 天内过期：
```